homeassistant/components/modbus/modbus.py

```diff
@@ -234,6 +235,22 @@ async def async_stop_hub(service: ServiceCall) -> None:
 
     async def async_restart_hub(service: ServiceCall) -> None:
         """Restart Modbus hub."""
+        async_create_issue(
+            hass,
+            DOMAIN,
+            "deprecated_restart",
+            breaks_in_ha_version="2024.10.0",
+            is_fixable=False,
+            severity=IssueSeverity.WARNING,
+            translation_key="deprecated_restart",
+            translation_placeholders={
+                "integration": DOMAIN,
+                "url": "https://www.home-assistant.io/integrations/modbus",
+            },
```

Comment:
You don't use this right now

Reply:
Removed (I thought it was always needed).